### PR TITLE
Refactor swagger docs

### PR DESCRIPTION
**Pull request by mederic-p**
# Overview

This PR introduces a refactoring of the 2 watcher api specs.
There was some inconsistencies in the documentation and some endpoints/params were outdated/not available.

The refactoring consists in a split into different files for better readability and maintainability.
When developing and updating the spec, these files should be updated and then the full spec should be re-generated based on these files.

For more details, you can check the [added documentation](https://github.com/omisego/elixir-omg/blob/refactor-swagger-doc/apps/omg_watcher/priv/swagger/swagger.md).

### CODE REVIEW

**Comment by kasima** [comment]
Once reviewed, I'd like to port this refactor into `master`.

File: apps/omg_rpc/priv/swagger/shared/responses.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+InternalServerError:
+  description: Returns an internal server error
+  content:
+    application/vnd.omisego.v1+json:
```
Comment (pnowosie) [inline apps/omg_rpc/priv/swagger/shared/responses.yaml]:
```suggestion
    application/json:
```

**Comment by pnowosie** [comment]
Wow, it was a lot of work, maybe not worth for such simple API... But the ability to test endpoint on `ari` is very appreciated :tada:  Thank you for taking the effort @mederic-p .
I've found some issues, we can address now or in following PR.

- [ ] there is smth wrong with operator API endpoints request schemata - they shows `required` as a field of the request object
__security critical endpoints__
- [ ] /account.get_utxos - request schema - missing required flag (red asterisk) on parameter
- [ ] /utxo.get_challenge_data  - request schema - missing required flag (red asterisk) on parameter
- [ ] /utxo.get_exit_data - same as above :point_up: 
- [ ] /transaction.submit - same as above :point_up: 
- [ ] /in_flight_exit.get_data - same as above :point_up: 
- [ ] /in_flight_exit.get_competitor - same as above :point_up: 
- [ ] /in_flight_exit.prove_canonical - same as above :point_up:

**Comment by mederic-p** [comment]
I'll update the PR and refactor the operator API too

**Comment by pnowosie** [comment]
Excellent work :exclamation:  Thank you :pray:

**Pull request merged**